homeassistant/helpers/entity_platform.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, *, hass, logger, domain, platform_name, platform,
             return
 
         # Async platforms do all updates in parallel by default
-        if hasattr(platform, 'async_setup_platform'):
+        if hasattr(platform, 'async_update'):
```

Comment:
There is no such thing as an `async_update` top level function ?

Reply:
Did you meant that 

https://github.com/home-assistant/home-assistant/blob/46ece3603f0d6ae15608649dab1635c0487e90af/homeassistant/helpers/entity.py#L376-L379

Reply:
`self` in the `entity.py` file will point at the `entity` object. `platform` points at the platform module.

Reply:
That is an Aha moment...